detection_rules/rule_validators.py

```diff
@@ -308,6 +317,30 @@ def validate_query_with_schema(self, data: 'QueryRuleData', schema: Union[ecs.Kq
             print(err_trailer)
             return exc
 
+    def validate_rule_type_configurations(self, data: EQLRuleData, meta: RuleMeta) -> \
+            Tuple[List[Optional[str]], bool]:
+        """Validate EQL rule type configurations."""
+        if data.timestamp_field or data.event_category_override or data.tiebreaker_field:
+
+            # get a list of rule type configuration fields
+            set_fields = [f for f in [
+                data.get("timestamp_field"),
+                data.get("event_category_override"),
+                data.get("tiebreaker_field")] if f]
```

Comment:
nit: May make this a bit more pythonic without the list comprehension.

```suggestion
            # Get a list of rule type configuration fields
            fields = ["timestamp_field", "event_category_override", "tiebreaker_field"]
            set_fields = list(filter(None, (data.get(field) for field in fields)))
```